.github/workflows/macos.yml

```
@@ -356,6 +366,12 @@ jobs:
           echo '::group::raw'; cat bld/lib/curl_config.h || true; echo '::endgroup::'
           cat bld/lib/curl_config.h | grep -F '#define' | sort || true
 
+      - name: 'build-cert'
+        if: contains(matrix.build.name, 'SecureTransport')
```

```suggestion
        if: contains(matrix.build.generate, '-DCURL_USE_SECTRANSP=ON')
```
sync condition with other step.